sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md

```diff
@@ -119,6 +134,99 @@ foreach (KnowledgeBaseAnswer answer in response.Value.Answers)
 }
 ```
 
+## Examples - QuestionAnsweringProjectsClient
+
+The following examples show common scenarios using the `QuestionAnsweringProjectsClient` instance [created in this section](#create-a-questionansweringprojectsclient).
+
+### Create a new project
+
+To create a new project, you must specify the project's name and a create a `RequestContent` instance with the parameters needed to set up the project.
+
+```C# Snippet:QuestionAnsweringProjectsClient_CreateProjectAsync
+// Set project name and request content parameters
+string newProjectName = "NewFAQ";
+RequestContent creationRequestContent = RequestContent.Create(
+    new
+    {
+        description = "This is the description for a test project",
+        language = "en",
+        multilingualResource = false,
+        settings = new
+        {
+            defaultAnswer = "No answer found for your question."
+        }
+    }
+    );
+
+Response creationResponse = await client.CreateProjectAsync(newProjectName, creationRequestContent);
+
+// Projects can be retrieved as follows
+AsyncPageable<BinaryData> projects = client.GetProjectsAsync();
+
+Console.WriteLine("Projects: ");
+await foreach (BinaryData project in projects)
+{
+    Console.WriteLine(project);
+}
+```
+
+### Deploy your project
+
+Your projects can be deployed using the `DeployProjectAsync` or the synchronous `DeployProject`. All you need to specify is the project's name and the deployment name that you wish to use. Please note that the service will not allow you to deploy empty projects.
+
+```C# Snippet:QuestionAnsweringProjectsClient_DeployProjectAsync
+// Set deployment name and start operation
+string newDeploymentName = "production";
+Operation<BinaryData> deploymentOperation = await client.DeployProjectAsync(waitForCompletion, newProjectName, newDeploymentName);
+
+Console.WriteLine($"Update Sources operation result: \n{deploymentOperation.Value}");
+
+// Deployments can be retrieved as follows
+AsyncPageable<BinaryData> deployments = client.GetDeploymentsAsync(newProjectName);
+Console.WriteLine("Deployments: ");
+await foreach (BinaryData deployment in deployments)
+{
+    Console.WriteLine(deployment);
+}
+```
+
+### Add a knowledge source
+
+One way to add content to your project is to add a knowledge source. The following example shows how you can set up a `RequestContent` instance to add a new knowledge source of the type "url".
+
+```C# Snippet:QuestionAnsweringProjectsClient_UpdateSourcesAsync
+// Set request content parameters for updating our new project's sources
+string sourceUri = "https://www.microsoft.com/en-in/software-download/faq";
+bool waitForCompletion = true;
```

Comment:
Seems a little odd to add a local variable for this. If you want to pass `true` but also make it obvious what the parameter is for, consider using named parameters below where you pass this e.g., 

```csharp
await client.UpdateSourcesAsync(waitForCompletion: true, newProjectName, updateSourcesRequestContent);
```

Reply:
Yea, this is a much better solution